src/sharepointService.ts

```diff
@@ -44,8 +44,26 @@ export function generateSharepointColumnResourceDefinitions(
               label: choice,
               value: choice,
             })),
-            checkIsFormElementSupported() {
-              return false
+            checkIsFormElementSupported(
+              formElementWithOptions,
+              { formsAppEnvironmentId, formElementOptionsSets },
+            ) {
+              const formElementOptionsSetId =
+                formElementWithOptions.dynamicOptionSetId
+              return (
+                !!formElementOptionsSetId &&
+                formElementWithOptions.optionsType === 'DYNAMIC' &&
+                formElementOptionsSets.some((formElementOptionsSet) => {
+                  return (
+                    formElementOptionsSet.type === 'SHAREPOINT_LIST_COLUMN' &&
+                    formElementOptionsSet.environments.some(
+                      (env) =>
+                        env.formsAppEnvironmentId === formsAppEnvironmentId &&
+                        env.sharepointColumn.id === columnDefinition.id,
```

Comment:
Should we be checking with the integration as well? Or we won't have access to it when this is used?

Reply:
We could pass it in, but if the user has drilled down to a point where they are loading columns (after selecting site and list), the column id **should** be unique enough even across entra applications i expect.